app/main.py

```diff
@@ -1,6 +1,30 @@
+import requests
+import os
+
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+API_KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    result = requests.get(
+        URL
+        + "?"
+        + f"key={API_KEY}"
+        + "&"
+        + f"q={FILTERING}"
+        + "&"
+        + "aqi=no"
```

Comment:
Looks so bad, 
FYI .get you can pass params as dict
Or write inline
```suggestion
        + f"{URL}?key={API_KEY}&q={FILTERING}&aqi=no"
```

Reply:
yeah, thx, mine version is something much over the top :)